dmcontent/questions.py

```diff
@@ -742,24 +745,29 @@ def answer_required(self):
             return False
 
         lookup_question_by_id = {q.id: q for q in self.questions}
-        followup_questions = set()
+        followups_optionals = set()
```

Comment:
Aren't we excluding optional sub-questions a couple of lines above? Why has this set been renamed?

Reply:
No, that excludes optional multiquestions, not subquestions. I changed the name because I realized that questions also get added to this set if they're not `.answer_required`, they don't have to be mentioned as another question's followup. Though I now realize that a question _not_ being `.answer_required` doesn't _necessarily_ make it "optional" - it could be already completed, so `followups_optionals` isn't technically accurate either.

Reply:
Ok better now.